api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERSemiAnnualReport.java

```diff
@@ -88,7 +88,7 @@ public List<ReportDesign> constructReportDesigns(ReportDefinition reportDefiniti
       reportDesign =
           createXlsReportDesign(
               reportDefinition,
-              "Template_PEPFAR_MER_2.6.1_Semiannual_Report_v1.5.2.xls",
+              "Template_TX TB_PEPFAR_MER_2.7_Semiannual_Report_v.0.3.xls",
```

Comment:
Is there a reason we are downgrading the version of the template from v1.5.3 to v0.3? @helderjosue

Reply:
The version was upgraded from **MER_2.6.1_Semiannual** to **MER_2.7_Semiannual**
And its actually a temporary template that will be used for txtb, the BA team is going to provide the final template once its approved @ItsMurumba

Reply:
I see, I had not seen the 2.7 part. Thank you for the clarification @helderjosue